LEAF_Request_Portal/form.php

```diff
@@ -3061,6 +3066,24 @@ public function query($inQuery)
             }
         }
 
+        if ($joinRecordResolutionBy) {
```

Comment:
Need this not to be ambiguous.  "If" what?  !empty?  TRUE?

Reply:
Yeah I think all the functions are coded in if statements like this unfortunately 🤣

Reply:
fixed